lms/static/js/jquery.gradebook.js

```diff
@@ -27,7 +27,10 @@ var Gradebook = function($element) {
 
 	var moveDrag = function(e) {
 		var offset = e.pageX - mouseOrigin;
-		var targetLeft = clamp(tableOrigin + offset, -maxScroll, 0);
+		// Updating direction of scroll by calculating pixels. O me left most and
```

Comment:
The working of this code is that, 
- Grade book container (parent) has fix width. 
- Grade book table has width depends on it column.
- So when user try to drag columns to left or right then in consequence I am scrolling table to the desire direction. 
- Second user can scroll left or right in parent container if he do not want dragging. 
- I also added tooltip on records

**Note: In past this grade book only support dragging to view left or right columns on viewport only if table's size was greater then parent size. They use to add margin  (add pixels on css property ```left: x```) to move to columns in right direction.
Not instead of adding margin I am using scroll for the purpose (in dragging) plus user can scroll horizontally with keyboard.**